test_suite/generic/helpers.py

```diff
@@ -0,0 +1,10 @@
+from test_suite import conftest
+
+
+def instance_data(host):
```

Comment:
I mentioned in this comment that since this fixture is limited by a class scope of "function", it is impossible to use it in other fixtures like in this case which are using class scope for example.

There are two solutions for this:
1. Change the existing fixture of instance_data to class scope
2. Create an independent function (like my solution in helpers.py) that can be used in any fixture or method and it is not limited by any class scope.

Reply:
Thanks, I think we can try with your proposal **n.1**, see if it works as expected, with multiple deployments.
If that does not behave correctly due to class instantiation in parallel testing scenarios, maybe we can just use `instance_data` fixture.